### ISSUE TITLE
Open telemetry Exemplars( Correlation between traces and metrics using java)  are still implementation stage or its completed 

### ISSUE THREAD

**Issue by ArjunYadav0082** [open-telemetry/opentelemetry-java-instrumentation#10012]
### Is your feature request related to a problem? Please describe.

Correlation between traces and metrics using java Exemplars Still implementation state 

### Describe the solution you'd like

Can you please share Examplars  examples

**Comment by trask** [comment]
yes, exemplars are implemented, here's an example: https://github.com/open-telemetry/opentelemetry-java-examples/blob/e3b4a3392eb2231753b13b3f250fcc80ef74135c/prometheus/README.md?plain=1#L42

**Comment by trask** [comment]
cc @jackshirazi @AlexanderWert
